### PR TITLE
Add fr-FR translation

### PR DESCRIPTION
**Pull request by Odhim**
Add French translation.

### CODE REVIEW

**Comment by WardBenjamin** [comment]
Looks great!

File: Assets/StreamingAssets/Localization/fr_FR.lang [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+selection_hp={0}/{1}
+build=Construct. {0}
+furn_steel_wall=Mur en acier
+furn_steel_wall_desc=Un mur, fait d'acier.
+furn_door=Porte
+furn_door_desc=Une porte.
+furn_stockpile=Stock
+furn_stockpile_desc=Un stockage, pour vos matériaux.
+furn_oxygen_generator=Générateur d'oxygène
+furn_oxygen_generator_desc=Génère l'oxugène pour vos astronautes.
```
Comment (JoN1oP) [inline Assets/StreamingAssets/Localization/fr_FR.lang]: l'oxugène -> l'oxygène

Comment (Odhim) [inline Assets/StreamingAssets/Localization/fr_FR.lang]: Change done, thanks for the highlight.

**Comment by RodRitter** [comment]
It looks like there is already a french translation committed #262 - If there is language improvement, you might want to resolve the merge conflict first.

**Pull request closed without merging**